packages/inference/test/tapes.json

```diff
@@ -207,7 +207,7 @@
       "statusText": "OK"
     }
   },
-  "ca042a4ac6a4f262adf83890752b75c22bab477f8ed65e469a87dbe8dcb4941a": {
+  "741f4bc66664a45c59af4ca4251b0604c297cd8927ea66d7fa138d5f6799c61e": {
     "url": "https://api-inference.huggingface.co/models/superb/hubert-large-superb-er",
     "init": {
       "headers": {
```

Comment:
I didn't thought about this issue, the `Authorization` header is saved in the tapes, we should fix the VCR, update the tapes and possibly revoke the exposed tokens (cc @coyotte508 )

Reply:
Oops I thought it's a special access token for testing :D
I can fix in separate PR

Reply:
Actually it's my token, I revoked it

Reply:
Fixed in #133